operators/src/source/ogr_source.rs

```diff
@@ -178,9 +180,11 @@ impl Default for OgrSourceTimeFormat {
 ///  - text: an array of column names containing alpha-numeric values
 ///  - rename: a. optional map of column names from data source to the name in the resulting collection
 #[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
+#[serde(rename_all = "camelCase")]
 pub struct OgrSourceColumnSpec {
     pub x: String,
     pub y: Option<String>,
+    pub csv_headers: Option<CsvHeaders>,
```

Comment:
Maybe we should CSV-specific stuff into some _ format-specific parameters_ > csv field?
We would have for each file type one struct then, starting with CSV.

Reply:
0fcb8508b434b9d310b9344204d08121dc741701